generator/Makefile

```diff
@@ -0,0 +1,5 @@
+golden:
+	go test github.com/googleapis/google-cloud-rust/generator/cmd/protoc-gen-gclient -update-golden
+
+test:
+	go test ./...
```

Comment:
nit: missing newline at the end of the file.

Reply:
done